src/org/traccar/protocol/Tk103ProtocolEncoder.java

```diff
@@ -97,6 +97,14 @@ protected Object encodeCommand(Command command) {
                     return formatCommand(command, "({%s}AV010)", Command.KEY_UNIQUE_ID);
                 case Command.TYPE_ENGINE_RESUME:
                     return formatCommand(command, "({%s}AV011)", Command.KEY_UNIQUE_ID);
+                case Command.TYPE_OUTPUT_CONTROL:
+                    if (command.getAttributes().containsKey(Command.KEY_DATA)) {
+                        if (command.getAttributes().get(Command.KEY_DATA).equals("1")) {
+                            return formatCommand(command, "({%s}AV001)", Command.KEY_UNIQUE_ID);
```

Comment:
I think it's better to just append `KEY_DATA` value to the command.

Reply:
ok, Changing it.